package.json

```diff
@@ -8,10 +8,12 @@
   "scripts": {
     "build:core": "cd packages/core && yarn build",
     "build:core:watch": "cd packages/core && yarn build:watch",
+    "build:core-storybook": "cd packages/core && yarn storybook:build",
     "build:react": "cd packages/react && yarn build",
     "build:react:watch": "cd packages/react && yarn build:watch",
     "build": "yarn build:core; yarn build:react",
-    "build:watch": "yarn build:core:watch & yarn build:react:watch"
+    "build:watch": "yarn build:core:watch & yarn build:react:watch",
+    "deploy": "firebase deploy --project=default"
```

Comment:
It could be great to have a section in README to explain the firebase deploy :-).

Reply:
Yes indeed, added.